styles.css

```diff
@@ -0,0 +1,158 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+body {
+  background-color: #f4f5f6;
+  font-family: 'Slabo 27px', serif;
+}
+
+.container {
+  background-color: white;
+  max-width: 1310px;
+}
+
+.bottom-nav {
+  border-bottom: 2px solid #f4f5f6;
+}
+
+.category,
+.category-2,
+.category-3,
+.category-4,
+.category-5,
+.category-6,
+.category-7 {
+  font-family: 'Roboto Condensed', sans-serif;
+  color: #f72210;
+  text-transform: uppercase;
+  background-color: rgba(255, 255, 255, 0.9);
+  bottom: 12.2em;
+}
+
+.category-3,
+.category-4 {
+  bottom: 9.2em;
+}
+
+.category-4,
+.category-5 {
+  color: #3581f4;
+}
+
+.category-5,
+.category-7 {
+  bottom: 12.2em;
+}
+
+.category-6 {
+  bottom: 5.9em;
+}
+
+.vs-img {
+  display: flex;
+  font: bold 13px/20px 'Roboto condensed', Arial, sans-serif;
+  padding: 8px 15px 9px;
+  text-transform: uppercase;
```

Comment:
[OPTIONAL]- You can use the bootstrap class `text-uppercase` for this. Feel free to use [this cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) for more information.